### PR TITLE
Fix retryLimit off by one

### PR DESCRIPTION
**Pull request by anson627**
based on tchannel change:
https://github.com/uber/tchannel-node/pull/16

r: @Raynos @ShanniLi @jcorbin

### CODE REVIEW

**Comment by anson627** [comment]
benchmark failed to start, seems to be irrelevant.

File: handler.js [hunk 1]
```diff
@@ -308,7 +308,6 @@ function sendRelay(opts, callback) {
                 headers: {
                     cn: self.callerName
                 },
-                retryLimit: 1,
```
Comment (Raynos) [inline handler.js]: retryLimit: 0 please. It defaults to 5.

**Comment by Raynos** [comment]
I will fix this myself; personally.

**Comment by Raynos** [comment]
@anson627 dont yolo merge without review.

**Comment by anson627** [comment]
@Raynos sorry, my bad.

**Pull request merged**